LongReadRNABenchmark/cupcake_docker/Dockerfile

```diff
@@ -51,6 +51,8 @@ WORKDIR $SRC
 
 COPY remove_fastq_duplicates.py $SRC/remove_fastq_duplicates.py
 
+ENV SKLEARN_ALLOW_DEPRECATED_SKLEARN_PACKAGE_INSTALL=True
```

Comment:
This feels like the sklearn version should also be pinned somewhere in the build, right?

Reply:
agreed, but the issue is actually that sklearn really shouldn't be installed, the correct pypi package is scikit-learn.  sklearn is kept around [to prevent malicious actors from using it](https://pypi.org/project/sklearn/).  As far as I can tell from that link, there is no version of sklearn that can be installed without setting this env variable.  The "right" way to fix this is to insall scikit-learn instead, but since sklearn is set as a requirement in the cupcake setup.py, I think this would require making a change to cupcake itself, which doesn't seem worth it since the tool itself is not longer maintained.

Reply:
I see, that makes sense. Maybe can you add a comment to the Dockerfile just quickly explaining this for future excavators? Or even just a link back to this comment would suffice